ci-operator/config/stolostron/governance-policy-template-sync/stolostron-governance-policy-template-sync-release-2.6.yaml

```
@@ -8,7 +8,7 @@ build_root:
   image_stream_tag:
     name: builder
     namespace: stolostron
-    tag: go1.17-linux
+    tag: go1.18-linux
 images:
 - dockerfile_path: build/Dockerfile
   from: base
```

Would you also add the `inputs` here?
```suggestion
  from: base
  inputs:
    root:
      as:
      - golang:1.18
```